invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -28,7 +28,9 @@
 def make_affiliation_index(attr, obj, dummy_ctx):
     """Serializes creators/contributors for easier UI consumption."""
     # Copy so we don't modify in place the existing dict.
-    creators = deepcopy(obj['metadata'][attr])
+    creators = deepcopy(obj['metadata'].get(attr))
+    if not creators:
+        return {}
```

Comment:
Minor:I think this should be returning missing instead?

```suggestion
        return missing
```

Reply:
Indeed! Fixed.